x-pack/plugins/reporting/server/types.ts

```diff
@@ -70,7 +68,7 @@ export interface ReportingStartDeps {
   licensing: LicensingPluginStart;
   taskManager: TaskManagerStartContract;
   screenshotting?: ScreenshottingStart;
-  security?: SecurityPluginStart;
+  security?: SecurityServiceStart;
```

Comment:
This field will just be needed for the `ReportingInternalStart` interface in `reporting/server/core.ts`
```suggestion
```